contracts/rollup/BatchManager.sol

```diff
@@ -143,11 +140,8 @@ contract BatchManager is Parameters {
             block.number > batches[batchID].finaliseOn(),
             "This batch is not yet finalised, check back soon!"
         );
-        require(
-            !Bitmap.isClaimed(batchID, withdrawalBitmap),
-            "Rollup: Already withdrawn"
-        );
-        Bitmap.setClaimed(batchID, withdrawalBitmap);
+        // delete the batch to mark the stake has been withdrawn
+        delete batches[batchID];
```

Comment:
We now track the withdrawal with the existence of the batch.
📈 simplicity/ gas saving on storage
📉 the logic is indirect

Reply:
Ah! Bad idea deleting batches. The vault still needs it.